umap/static/umap/js/modules/utils.js

```diff
@@ -242,10 +242,12 @@ export function greedyTemplate(str, data, ignore) {
       }
       for (let i = 0; i < vars.length; i++) {
```

Comment:
```suggestion
      for (const path of vars) {
```

Reply:
Can you also remove `path = xxx` in the patch or should I ?